Dockerfile

```diff
@@ -11,7 +11,10 @@ FROM alpine:3.18.0
 
 WORKDIR /root/
 
-RUN apk add --no-cache --update bash libmnl gcompat iptables openresolv iproute2
+RUN apk add --no-cache --update bash libmnl gcompat openresolv iproute2
+RUN apk add iptables ip6tables \
+    && mv -v /sbin/ip6tables /sbin/ip6tables-disabled \
```

Comment:
failed to intialize firewall:  couldn't get ipv6 filter table chains, error: running [/sbin/ip6tables -t filter -S --wait]: exit status 3: modprobe: can't change directory to '/lib/modules': No such file or directory
ip6tables v1.8.8 (legacy): can't initialize ip6tables table filter': Table does not exist (do you need to insmod?)
Perhaps ip6tables or your kernel needs to be upgraded.

it's an issue with ip6tables legacy version, this is the workaround to make it work...

Reply:
there is one more work around for this issue  https://ilhicas.com/2018/04/08/Fixing-do-you-need-insmod.html
you need to run this command on the host: `sudo modprobe ip6table_filter`

Reply:
> failed to intialize firewall: couldn't get ipv6 filter table chains, error: running [/sbin/ip6tables -t filter -S --wait]: exit status 3: modprobe: can't change directory to '/lib/modules': No such file or directory ip6tables v1.8.8 (legacy): can't initialize ip6tables table filter': Table does not exist (do you need to insmod?) Perhaps ip6tables or your kernel needs to be upgraded.
> 
> it's an issue with ip6tables legacy version, this is the workaround to make it work...

so why not just install iptables-nft instead of iptables/ip6tables?

Reply:
the iptables library https://github.com/coreos/go-iptables wil only work `iptables/iptables6` binaries